<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="MainFrame">
     <property name="styleSheet">
      <string notr="true">QFrame#MainFrame {
	background-color: rbg(0,0,0)
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QFrame" name="ContentFrame">
        <property name="styleSheet">
         <string notr="true">QFrame#ContentFrame{
 	background-color: rgb(255,255,255);
	border-radius: 5px;
	border-color: rgb(107, 108, 108);
	border-width: 1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="bottomMargin">
          <number>12</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="exitPushButton">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    color: #ffffff;
    background-color: #ffc107;
    border-width: 1px;
	color: rgb(96, 97, 97);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color :rgb(96, 97, 97);
	background-color: #ffffff;
    border-width: 1px;
    border-color: : rgb(96, 97, 97);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
             </property>
             <property name="text">
              <string>X</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="LeftContentFrame">
               <property name="minimumSize">
                <size>
                 <width>250</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>250</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
                <item>
                 <widget class="QLineEdit" name="TotalPaidLineEdit">
                  <property name="minimumSize">
                   <size>
                    <width>200</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>897987</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="font">
                   <font>
                    <family>Futura</family>
                    <pointsize>45</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLineEdit{
color:#ffc107;
padding: 3px;
padding-left: 5px;
padding-right: 5px;
background-color: rgb(96, 97, 97);
}

QLineEdit:focus
{
    color: rgb(96, 97, 97);
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
	background-color: #ffc107;
}</string>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="PaymentButtonVerticalLayout">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="TotalLabel">
               <property name="font">
                <font>
                 <family>Futura</family>
                 <pointsize>64</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>Rp 0.0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QScrollArea" name="PaymentScrollArea">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255,255,255);</string>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOn</enum>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="scrollAreaWidgetContents">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>448</width>
                  <height>316</height>
                 </rect>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <property name="leftMargin">
                  <number>2</number>
                 </property>
                 <property name="topMargin">
                  <number>2</number>
                 </property>
                 <property name="rightMargin">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>2</number>
                 </property>
                 <item>
                  <layout class="QVBoxLayout" name="PaymentVerticalLayout">
                   <property name="spacing">
                    <number>2</number>
                   </property>
                   <property name="sizeConstraint">
                    <enum>QLayout::SetDefaultConstraint</enum>
                   </property>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>30</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>Due</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="TotalDueLabel">
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>30</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="text">
                  <string>0.0</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>30</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Change</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="font">
                  <font>
                   <family>Futura</family>
                   <pointsize>30</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="CancelPushButton">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Futura</family>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    color: #ffffff;
    background-color: #ffc107;
    border-width: 1px;
	color: rgb(96, 97, 97);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color :rgb(96, 97, 97);
	background-color: #ffffff;
    border-width: 1px;
    border-color: : rgb(96, 97, 97);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
             </property>
             <property name="text">
              <string>CANCEL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="PaidPushButton">
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>40</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>40</width>
               <height>60</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Futura</family>
               <pointsize>22</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
    color: #ffffff;
    background-color: #ffc107;
    border-width: 1px;
	color: rgb(96, 97, 97);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:pressed{
    color :rgb(96, 97, 97);
	background-color: #ffffff;
    border-width: 1px;
    border-color: : rgb(96, 97, 97);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}
QPushButton:disabled{
    color: rgb(82, 96, 117);
    background-color: rgb(0, 0, 0);
    border-width: 1px;
    border-color: rgb(82, 96, 117);
    border-style: solid;
    border-radius: 3;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
             </property>
             <property name="text">
              <string>PAID</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
